resources/js/Pages/Partials/SinkDocs.vue

```diff
@@ -0,0 +1,20 @@
+<script setup>
+import { ref, onMounted } from 'vue';
+
+const props = defineProps({
+    sinkId: { type: String, required: true },
+});
+
+const docs = ref();
+
+onMounted(async () => {
+    const result = await axios.get(`/api/sinks/${props.sinkId}/getDoc`);
+    docs.value = result.status === 200 ? result.data : 'No documentation provided';
+});
+</script>
+
+<template>
+  <!-- eslint-disable vue/no-v-html -->
+  <div v-html="docs" />
+  <!-- eslint-enable -->
```

Comment:
Trenger du å slå på eslint igjen? Gjelder den ikke bare for linja under? og gjelder den ikke bare for 'vue/no-v-html'? Hvis den må 'slås på' igjen så tenker jeg også at det er ikke nødvendig. Fila er liten og unntaket kan fint gjelde hele den.

Reply:
fixed